files/src/config/module-map.ts

```diff
@@ -1,6 +0,0 @@
-/**
```

Comment:
We may want to keep this around, but move it up into the root.

Without it, I believe that VSCode and other TS aware editors will show a warning that this module cannot be found...

Thoughts @dgeb / @mmun  / @tomdale?